tso-server/src/main/java/org/apache/omid/tso/RetryProcessorImpl.java

```diff
@@ -69,7 +72,8 @@
     private final Meter noCTFoundMeter;
 
     @Inject
-    RetryProcessorImpl(MetricsRegistry metrics,
+    RetryProcessorImpl(@Named("PersistenceStrategy") WaitStrategy strategy,
```

Comment:
Typo in named. Should be the same name that is in the DisruptorModule.java for the retry processor:

```
@Named("RetryStrategy")
```